dequorum/templates/dequorum/thread_list.html

```diff
@@ -16,11 +16,11 @@
     </table>
     </form>
 {% for thread in threads %}
-    <div class="thread">
+    <div class="{% if forloop.first %}topborder {% endif %}{% cycle 'thread1' 'thread2' %}">
```

Comment:
This can also be done as:
    {{ forloop.first|yesno:"topborder " }}

But would probably be done best using :first-child or :first-of-type pseudo selectors